File: Assets/younne/Prac/Script/TipsView.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.UI;
using XLua;

namespace Younne
{
    [Hotfix]
    [LuaCallCSharp]
    public class TipsView : MonoBehaviour
    {
        public Text tipsText;

        //[Hotfix]
        public void SetTips()
        {
            tipsText.text = "Old";
        }
    }
}

File: Assets/younne/Prac/Lua/Hotfix/TipsView.lua

local TipsView = {}

function TipsView.Hotfix( )
	local tipsView = CS.Younne.TipsView
	xlua.hotfix(tipsView, "SetTips", 
		function ( ... )
			tipsView.tipsText = 'New'
		end
	)
end


TipsView.Hotfix()

return TipsView